src/main/java/org/sasanlabs/service/vulnerability/jwt/JWTVulnerability.java

```diff
@@ -128,8 +128,7 @@ public JWTVulnerability(IJWTTokenGenerator libBasedJWTGenerator, IJWTValidator j
             value = LevelConstants.LEVEL_1,
             descriptionLabel = "URL_CONTAINING_JWT_TOKEN",
             htmlTemplate = "LEVEL_1/JWT_Level1",
-            parameterName = JWT,
-            sampleValues = {PRECOMPUTED_JWT_HS256_HIGH_STRENGTH})
```

Comment:
can you please remove these constants too?

Reply:
There are 4 precomputed constants.

Reply:
do you wish to remove all 4 constants (line 128 to 131) or to just remove parameterName from this JWTVulnerability.java file?

Reply:
constants only as they are not refered anywhere else now. Not parameter name

Reply:
got it now, thanks for the clarification!